src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityFilter.java

```diff
@@ -19,13 +20,13 @@
 
 public enum EntityFilter {
 	ALL(EntityType.values()),
-	ALL_DIRECTORY(project, folder, file, link),
+	ALL_BUT_LINK(project, folder, file, table, entityview, dockerrepo, submissionview),
+	PROJECT_FOLDER_FILE_LINK(project, folder, file, link),
```

Comment:
This is what's displayed in the EntityTree by default. It used to be called `ALL`, but it could still use a better name than this.